src/github.js

```diff
@@ -964,6 +964,22 @@
                body: comment
             }, cb);
          };
+         this.edit = function (issue,options,cb) {
+             if (typeof issue === 'object') {
+                 issue = issue.number;
+             };
+             _request('PATCH', path + '/' + issue, options, function(err, res) {
```

Comment:
Here you can pass `cb` as the last argument. There is no need to define an anonymous function.